RecoTracker/SpecialSeedGenerators/python/SimpleCosmicBONSeeder_cfi.py

```diff
@@ -39,7 +39,7 @@ def makeSimpleCosmicSeedLayers(*layers):
             MaxNumberOfStripClusters = cms.uint32(300),
```

Comment:
casually looking in the online DQM configuration for the strip client, I noticed:

https://github.com/cms-sw/cmssw/blob/1e14eab3d2f0372afcbc497bc85b4f2d3a301648/DQM/Integration/python/clients/sistrip_dqm_sourceclient-live_cfg.py#L271-L272

perhaps it should be considered in the offline reconstruction as well.